website_sale_commission/models/sale_order_line.py

```diff
@@ -44,3 +44,11 @@ def unlink(self):
                     o.team_id.team_type == 'website'):
                 order.update_website_sale_commission()
         return res
+
+    # Override _get_display_price() to avoid Odoo from displaying the list
+    # price of the commission product on Website, the display price should
+    # also be controlled by the update_website_sale_commission()
+    @api.multi
+    def _get_display_price(self, product):
+        price = super(SaleOrderLine, self)._get_display_price(product)
+        return self.price_unit if self.is_commission else price
```

Comment:
@TimLai125 I think it'd be better to make it like below to save some computation. A minor point.
```.py
        if self.is_commission:
            return self.price_total
        else:
            return super(SaleOrderLine, self)._get_display_price(product)
```